portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,17 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>Carine Gordillo</p>
+      <p> Hello! I'm Carine  and I live in Downey, California but parents are originally from<br/> Zapopan, Jalisco. Some of my favorite hobbies are hiking, being with my family and
+      <br/> taking on new projects. I am 19 years old and attend Cal State Long Beach. Currently, I am <br/> participating in a progam by Google called SPS and I am on team 41! </p>
+      <img src="/images/cat.jpg" width="400"/>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
+      <p>Click here to get a random quote:</p>
       <button onclick="addRandomGreeting()">Hello</button>
```

Comment:
Optional: I know quotes could be greetings, but I recommend updating the existing code to more accurately reflect what the code is doing. So instead of "addRandomGreeting()" maybe it becomes "addRandomQuote()" or "getRandomQuote()"